.github/workflows/build_linux_wheels.yaml

```diff
@@ -0,0 +1,175 @@
+    with:
+      package-type: ${{ needs.setup-variables.outputs.PACKAGE_TYPE }}
+      os: linux
+      test-infra-repository: ${{ needs.setup-variables.outputs.TEST_INFRA_REPOSITORY }}
+      test-infra-ref: ${{ needs.setup-variables.outputs.TEST_INFRA_REF }}
+      with-cuda: disable
+      with-rocm: disable
+  build:
+    needs:
+      - setup-variables
+      - generate-matrix
+    strategy:
+      fail-fast: false
+      matrix: ${{ fromJSON(needs.generate-matrix.outputs.matrix) }}
+    env:
+      PYTHON_VERSION: ${{ matrix.python_version }}
+      CU_VERSION: ${{ matrix.desired_cuda }}
+      UPLOAD_TO_BASE_BUCKET: ${{ matrix.upload_to_base_bucket }}
+    name: ${{ matrix.build_name }}
+    runs-on: ${{ matrix.validation_runner }}
+    container:
+      image: ${{ matrix.container_image }}
+      options: ${{ matrix.gpu_arch_type == 'cuda' && '--gpus all' || ' ' }}
+    # If a build is taking longer than 60 minutes on these runners we need
+    # to have a conversation
+    timeout-minutes: 120
+    steps:
+      - name: Clean workspace
+        shell: bash -l {0}
+        run: |
+          set -euxo pipefail
+          echo "::group::Cleanup debug output"
+          rm -rf "${GITHUB_WORKSPACE}"
+          mkdir -p "${GITHUB_WORKSPACE}"
+          echo "::endgroup::"
+      - uses: actions/checkout@v3
+        with:
+          repository: ${{ needs.setup-variables.outputs.TEST_INFRA_REPOSITORY }}
+          ref: ${{ needs.setup-variables.outputs.TEST_INFRA_REF }}
+          path: test-infra
+      - uses: pytorch/test-infra/.github/actions/set-channel@main
+      - name: Set PYTORCH_VERSION
+        if: ${{ env.CHANNEL == 'test' }}
+        run: |
+          # When building RC, set the version to be the current candidate version,
+          # otherwise, leave it alone so nightly will pick up the latest
+          echo "PYTORCH_VERSION=${{ matrix.stable_version }}" >> "${GITHUB_ENV}"
+      - uses: pytorch/test-infra/.github/actions/setup-binary-builds@main
+        env:
+          PLATFORM: ${{ env.ARCH == 'aarch64'  && 'linux-aarch64' || ''}}
+        with:
+          repository: ${{ needs.setup-variables.outputs.REPOSITORY }}
+          ref: ${{ needs.setup-variables.outputs.REF }}
+          setup-miniconda: true
+          python-version: ${{ env.PYTHON_VERSION }}
+          cuda-version: ${{ env.CU_VERSION }}
+          arch: ${{ needs.setup-variables.outputs.ARCH }}
+      - name: Install torch dependency
+        run: |
+          set -euxo pipefail
+          # shellcheck disable=SC1090
+          source "${BUILD_ENV_FILE}"
+          # shellcheck disable=SC2086
+          ${CONDA_RUN} ${PIP_INSTALL_TORCH}
+      - name: Install python build dependency
+        run: |
+          set -euxo pipefail
+          # shellcheck disable=SC1090
+          source "${BUILD_ENV_FILE}"
+          # shellcheck disable=SC2086
+          ${CONDA_RUN} python -m pip install build
```

Comment:
So this will be supplanted by https://github.com/pytorch/test-infra/pull/5071/files? Will we be able to remove some of these other steps as well to rely directly on the test-infra version?